ppr-ui/tests/unit/ConfirmAmendment.spec.ts

```diff
@@ -48,6 +50,13 @@ describe('Confirm Amendment registration component', () => {
   const { assign } = window.location
   sessionStorage.setItem('KEYCLOAK_TOKEN', 'token')
 
+  beforeAll(async () => {
+    // Mimicks loading the data in the store in the previous step.
```

Comment:
Because we longer load data in the review step, I need to mimic loading it in the previous step.